.github/workflows/docs-wiki.yml

```
@@ -0,0 +1,62 @@
+name: Docs Wiki (generate + gate)
+
+# Regenerates the auto-generated API reference wiki from the source XML doc comments, fails if
+# the committed wiki drifted from the source, and compile-checks every ```csharp example in the
+# website docs + wiki against the real AiDotNet assemblies. The wiki lives only in the Astro
+# website (the single canonical docs site); the legacy Jekyll docs/ tree was removed.
+on:
+  push:
+    branches: [ master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)